.travis.yml

```diff
@@ -16,6 +16,7 @@ env:
   global:
     - JRUBY_OPTS='-J-XX:+TieredCompilation -J-XX:TieredStopAtLevel=1 -J-Djruby.compile.mode=OFF -J-Djruby.compile.invokedynamic=false'
 gemfile:
-  - gemfiles/Gemfile.actionpack5.0.0
+  - gemfiles/Gemfile.actionpack5.1
```

Comment:
This mostly duplicates what we have in the default `Gemfile` now. It has been a (rather loose) standard to move the `Gemfile` to the latest version and explicitly provide separate gemfiles for those versions we wish to track compatibility for going forward.

Probably worth just loosening the dependency in the default `Gemfile` in which case, no?